server/src/main/java/org/cloudfoundry/identity/uaa/scim/jdbc/JdbcScimUserProvisioning.java

```diff
@@ -196,6 +196,9 @@ public ScimUser create(final ScimUser user, String zoneId) {
         if (!hasText(user.getOrigin())) {
             user.setOrigin(OriginKeys.UAA);
         }
+        if (isCheckOriginEnabled(zoneId)) {
```

Comment:
This here is in createUser, where we have some lines before this call validateUserLimit , 
Both potentially perform a SQL to resolve the identityzone. Can we combine it and do the  identityZoneManager.getCurrentIdentityZone check once?

Reply:
fixed with [d9f8f25](https://github.com/cloudfoundry/uaa/pull/2688/commits/d9f8f25bfb0d791fcb5dfd8ea1d5c1e432c72428)